python/lsst/dax/imgserv/api_soda.py

```diff
@@ -0,0 +1,168 @@
+# LSST Data Management System
+# Copyright 2017-2019 AURA/LSST.
```

Comment:
This copyright is wrong. This seems like a new file so where is the 2017 coming from? Also the copyright is presumably SLAC not AURA. See the copyright guidelines in the developer guide.

Reply:
Fixed per guidelines.